packages/nextjs/pages/builders/0x55fF7E28e7cd43C104dC89Aa69285E0E6EBa064e.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+
+export default function yassirakhBuilderPage() {
```

Comment:
Nitpick: It's a good convention to start the name with capital letter 

Although it doesn't cause any errors in this NextJs pages context, but it would have thrown error / not worked if it was React component checkout this [react docs pitfall](https://react.dev/learn/your-first-component#step-2-define-the-function)

Reply:
You can also follow the convention for the project:

have a `const` here and export default at the bottom of the page.